app/connectors/Vrn.scala

```diff
@@ -0,0 +1,59 @@
+/*
```

Comment:
I think this file is in the wrong package. Shoudl be in models or connectors/models

Reply:
models, not connectors/models pls